napari/_qt/widgets/_tests/test_qt_play.py

```diff
@@ -122,29 +122,21 @@ def test_play_raises_index_errors(qtbot, ref_view):
     # play axis is out of range
     with pytest.raises(IndexError):
         view.dims.play(4, 20)
-        qtbot.wait(20)
```

Comment:
These raise when `AnimationWorker` is created or before, which is before the thread actually starts. So no need to wait or stop.

Reply:
Maybe stop so that we tear the down test safely? Or should that be done elsewhere?